apps/omg_db/mix.exs

```diff
@@ -32,6 +32,8 @@ defmodule OMG.DB.MixProject do
     [
       {:appsignal, "~> 1.0"},
       # version caused by dependency in merkle_patricia_tree from blockchain
+      {:rocksdb,
```

Comment:
will get swapped out once https://gitlab.com/barrel-db/erlang-rocksdb/merge_requests/112#note_167748150 gets merged!

Reply:
notice the comment above is about `:exleveldb` (I'm not sure if it is relevant anymore)